src/web/app/pages-help/instructor-help-page/instructor-help-getting-started/instructor-help-getting-started.component.html

```diff
@@ -0,0 +1,213 @@
+<h1 id="top" class="color-orange">Getting Started</h1>
+<div id="contentHolder">
+  <p>
+    Welcome to TEAMMATES!
+  </p>
+  <p>
+    To get started using TEAMMATES, follow the following steps, or watch our <a href="https://www.youtube.com/embed/mDtfmNmRwBM?autoplay=1&rel=0" target="_blank"><i class="fas fa-film"></i> Video Tour</a>.<br>
+    For more help, browse the answers to some <a routerLink="/web/front/help/instructor">frequently asked questions</a>.
+  </p>
+  <ol>
+    <li><a (click)="jumpTo('course-setup')" href>Set up a course</a></li>
```

Comment:
Angular's way is `[routerLink]=""` rather than `href` (which is equivalent to `href=""`). ([reference](https://stackoverflow.com/questions/35639174/passive-link-in-angular-2-a-href-equivalent))